### PR TITLE
Update "What network bandwidth does Lambda GPU Cloud provide?"

### PR DESCRIPTION
**Pull request by cbrownstein-lambda**
Closes: #47

### CODE REVIEW

File: content/en/cloud/network-bandwidth.md [hunk 1]
```diff
@@ -7,15 +7,51 @@ aliases:
...
-upgrading this connection to 20 Gbps in the coming months.
+## Texas, USA region (us-south-1)
+
+All of the instances in
+our Texas, USA region (us-south-1) share two symmetric 10 Gbps connections to
+the Internet, providing both redundancy and throughput of up to 20 Gbps.
 
 Internally, many of our servers have a 200 Gbps connection to our data
 center's network. Some of our servers have a 20 Gbps connection, shared by up
 to 8 instances.
```
Comment (cbrownstein-lambda) [inline content/en/cloud/network-bandwidth.md:18]: Can this be revised to match how the info for us-west-2 and us-east-1 is presented? (The bandwidth figures, below, probably need to be corrected.)

```suggestion
The bandwidth between instances in our Texas, USA region (us-south-1) is          
approximately 200 Gbps.                                                       
                                                                              
Bandwidth to the Internet is approximately 20 Gbps.
```

File: content/en/cloud/network-bandwidth.md [hunk 2]
```diff
@@ -7,15 +7,51 @@ aliases:
...
 
+## Arizona, USA region (us-west-2)
+
```
Comment (achurak) [inline content/en/cloud/network-bandwidth.md:21]: I'd probably skip the private vs public comparison, the difference is negligent, though I'd mention a recommendation to use private IPs for instance communication inside the same region and public IPs for instances in different regions.

File: content/en/cloud/network-bandwidth.md (hunk 1 above)
Comment (achurak) [reply content/en/cloud/network-bandwidth.md:18]: Definitely can't say that (`approximately 200gbps`). At the very least it has to be `up to` 200gbps.

Comment (cbrownstein-lambda) [reply content/en/cloud/network-bandwidth.md:18]: Okay. I can commit the suggestion then change _approximately_ to _up to_.

Also, I have the note saying _Real-world network bandwidth depends on a variety of factors, including the total number of connections opened by your applications._ I think that adequately informs that the numbers we provide _can_ be attained but that's definitely not guaranteed.

File: content/en/cloud/network-bandwidth.md (hunk 2 above)
Comment (cbrownstein-lambda) [reply content/en/cloud/network-bandwidth.md:18]: I created #168 to add your recommendation.

**Pull request merged**